### DISCUSSION TITLE
How to get templates with time and author name?

### DISCUSSION THREAD

**Discussion by ChandraSekharaSSHH** [cpeditor/cpeditor#726] [Q&A] [answered]
Can someone help me how to get a template that in header has time and author name . 
OS:Windows10
CPEditor:6.6.4

**Comment by ouuan** [comment, marked as answer]
1. This is a feature of CP Editor 6.7+, so you need to upgrade first.
2. The documentation is [here](https://cpeditor.org/v6.7/docs/preferences/extensions/#head-comments).
3. You can use `${time}` in this setting for the time, and add a line like "Author: someone" for the author.

**Reply by ChandraSekharaSSHH** [reply]
thanks. this worked
